verifyml/model_tests/FEAT/SHAPFeatureImportance.py

```diff
@@ -80,7 +81,10 @@ def get_shap_values(
                 model_output="margin",
                 feature_perturbation="tree_path_dependent",
             )
-            self.shap_values = explainer.shap_values(x_test_encoded)[1]
+            if is_classifier(model):
```

Comment:
More common to do

```py
self.shap_values = explainer.shap_values(x_test_encoded)[1] if is_classifier(model) else explainer.shap_values(x_test_encoded)
```
But also fine to leave it as it is if you find the line will become too long. Same for a few other instances of this in other files

Reply:
thanks for the checks!